packages/postcss-svgo/src/index.js

```diff
@@ -5,6 +5,7 @@ import { encode, decode } from './lib/url';
 const PLUGIN = 'postcss-svgo';
 const dataURI = /data:image\/svg\+xml(;((charset=)?utf-8|base64))?,/i;
 const dataURIBase64 = /data:image\/svg\+xml;base64,/i;
+const escapedQuotes = /\b([\w-]+)\s*=\s*\\"([\S\s]+?)\\"/g;
```

Comment:
Could you add a comment to say how this regular expression works?

Reply:
The regex will globally match:
`\b([\w-]+)` a word (a sequence of one or more [alphanumeric|underscore|dash] characters; followed by
`\s*=\s*` an equal sign character (=) between optional whitespaces; followed by
`\\"([\S\s]+?)\\"` any characters (including whitespaces and newlines) between literal escaped quotes (\")

Well, what it does is basically tries to find any variation of:
- `attr-foo=\"bar\"`
- `attr-foo   =\"bar\"`
- `attr-foo=   \"bar\"`
- `attr-foo   =   \"bar\"`
- `attr-foo=\"   bar\"`
- `attr-foo=\"bar   \"`
- `attr-foo=\"   bar   \"`
- etc

And normalize them to:
`attr-foo="  bar"`, or `attr-foo="bar   "`, or `attr-foo="  bar  "` (preserving whitespaces in between quotes.

I've added the comment in the code too, I hope it's not too verbose...

Reply:
Thank you, I think it is fine.